easybuild/easyconfigs/n/ncurses/ncurses-6.0-GNU-4.9.3-2.25.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'ncurses'
+version = '6.0'
```

Comment:
they are, ok, then a resync with `develop` should refresh this when #1991 goes in